poker/strategy.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, strategy, source='<string>'):
         for name in self._config.sections():
             # configparser set non-specified values to '', we want default to None
             values = dict.fromkeys(_Situation.__slots__, None)
+            if '__weakref__' in values:
+                del values['__weakref__']
             for key, val in self._config[name].items():
```

Comment:
Why is this needed?

Reply:
Without this change py.test fails
===================================================== ERRORS ======================================================
_________________________________ ERROR collecting tests/strategy/test_strategy.py _________________________________
tests/strategy/test_strategy.py:12: in <module>
    strategy = Strategy.from_file(filedir / 'push.strategy')
poker/strategy.py:64: in from_file
    return cls(strategy, source=filename)
poker/strategy.py:56: in __init__
    self._situations[name] = _Situation(**values)
E   TypeError: __init__() got an unexpected keyword argument '__weakref__'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================= 1 error in 0.48 seconds =========================================